### ISSUE TITLE
Don't fail when a DB is present and has the right schema

### ISSUE THREAD

**Issue by marco-c** [mozilla/bugbug#1131]
STR:
1) Bump the commits DB version:
2) Run the commit retriever script;
3) Run the bug retriever script.

The bug retriever script fails as the commits DB is not yet available on Taskcluster.

When we fix this, we can remove the workaround from 54f9d3ae8c2de90e84310d3ad2ad3fb18686b7f1.

**Comment by chidauri** [comment]
@marco-c If i understood it correctly, commit retriever mines commits and saves in the db(new version), so bug retriever should use that db if the new version isn't found on taskcluster, right?

**Comment by marco-c** [comment]
@chidauri yes, the problem is that db.py is checking if there is a new version of the DB on Taskcluster, but the new version is not available yet on Taskcluster if we've just bumped the version.

**Comment by chidauri** [comment]
@marco-c Is it a good idea to create a new `...version.local` file to differentiate that the db has been created on that system and so not available on taskcluster.

**Comment by marco-c** [comment]
We could probably just use the current `XXX.version` file, as it's not really being used.
